sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -77,7 +85,9 @@ def do_send_to_spark(self, cell, input_variable_name, var_type, output_variable_
             self.ipython_display.write(u'Successfully passed \'{}\' as \'{}\' to Spark'
                                        u' kernel'.format(input_variable_name, output_variable_name))
 
-    def execute_spark(self, cell, output_var, samplemethod, maxrows, samplefraction, session_name, coerce):
+    def execute_spark(self, cell, output_var, samplemethod, maxrows, samplefraction, session_name, coerce, output_handler=None):
+        _output_handler = output_handler or self.default_spark_output_handler 
```

Comment:
Do we use this `_` prefix pattern for parameters that have a corresponding default set on `self` elsewhere in the codebase? If not I would keep the name the same (`output_handler`) for consistency